### DISCUSSION TITLE
My deployed react app shows a blank white screen after loading initially

### DISCUSSION THREAD

**Discussion by B1ckb0x** [vercel/community#6954] [Help]
### Summary

I'm trying to work with Vercel. I deployed a React app, and I see the website in Vercel preview menu. However, when I load the website, it loads the page initially, and then a blank white screen appears.

**GitHub Repository**: https://github.com/B1ckb0x/frontend
**Website link**: https://frontend-ten-swart-80.vercel.app/

I have tried using the homepage:'.' and many more online discusiion, all to no avail. Any assistance in resolving this would be greatly appreciated. Thank you. 

**Vercel Preview menu:**
![image](https://github.com/vercel/community/assets/114180125/62cefd05-4cd2-4f22-a946-1cafe5998176)

**Website page:**
![image](https://github.com/vercel/community/assets/114180125/1b3f2431-c63f-4ed5-8221-432f97201063)

**What I see when I inspect the page:**
![image](https://github.com/vercel/community/assets/114180125/94435377-eb47-446b-a7af-6d0aafe48461)

### Example

https://frontend-ten-swart-80.vercel.app/

### Steps to Reproduce

1. Open the webpage and would see the app initially
2. After the website loads it displays a white blank screen

**Comment by sahilatahar** [comment]
There is no problem. It is working perfectly
